src/scripts/play.js

```diff
@@ -0,0 +1,207 @@
+const getRange = (startNum, endNum, step = 1) => {
+  let result = [];
+  for (let i = startNum; i <= endNum; i += step) {
+    result.push(i);
+  }
+  return result;
+};
+
+const randomChoice = (array) => {
+  return array[Math.floor(Math.random() * array.length)];
+};
+
+const Game = () => {
+  const backgroundSound = document.getElementById("background-sound");
+  const correctAnswerSound = document.getElementById("correct-answer");
+  const incorrectAnswerSound = document.getElementById("incorrect-answer");
+  const soundButton = document.getElementById("sound");
+  const gameScreen = document.querySelector(".game-screen");
+  const display = document.querySelector(".display-input");
+  const keyboard = document.querySelector(".keyboard");
+  let scoreOnDisplay = document.querySelector(".score-number");
+
+  soundButton.addEventListener("click", (event) => {
+    if (backgroundSound.paused) {
+      backgroundSound.play();
+    } else {
+      backgroundSound.pause();
+    }
+  });
+
+  function changeDisplay(number) {
+    if (display.value.length < 3) {
+      if (display.value == 0) {
+        display.value = number;
+      } else {
+        display.value += number;
+      }
+    }
+  }
+
+  const clearDisplay = (operation) => {
+    if (display.value !== 0) {
```

Comment:
If you need non-false check, you can check like this:

`if(display.value) {}`

Everywhere